.readthedocs-conda.yml

```diff
@@ -12,5 +12,5 @@ dependencies:
    - numpy
    - pillow
    - git+https://github.com/aflc/pyter@857a1552443f139a3
-   - https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0-cp35-cp35m-linux_x86_64.whl
+   - tensorflow
```

Comment:
Tenhle soubor instaluje prerekvizity na readthedocs. Tady je `>=0.1.0` skutečně redundantní protože se to instaluje při každým pushi načisto. A navíc to nikdo nečte, a tak to neslouží ani jako dokumentace.